packages/react-components/react-utilities/src/compose/getSlotsNext.test.tsx

```diff
@@ -1,45 +1,68 @@
 import * as React from 'react';
 import { getSlotsNext } from './getSlotsNext';
-import type { Slot } from './types';
+import type { ExtractSlotProps, Slot, SlotComponent, UnknownSlotProps } from './types';
 import { resolveShorthand } from './resolveShorthand';
-import { SLOT_RENDER_FUNCTION_SYMBOL } from './constants';
+import { SLOT_COMPONENT_METADATA_SYMBOL } from './constants';
+
+const resolveShorthandMock = <Props extends UnknownSlotProps>(props: Props): SlotComponent<Props> => {
+  // casting is required here as SlotComponent is a callable
+  return { [SLOT_COMPONENT_METADATA_SYMBOL]: { elementType: 'div' }, ...props } as SlotComponent<Props>;
+};
```

Comment:
Is there sense to update this test at all? `getSlotsNext` does nothing with `SLOT_COMPONENT_METADATA_SYMBOL`

Reply:
Yes, although there's no modifications in `getSlots` there's a modification in the `resolveShorthand` internals, and this test suite depends on `resolveShorthand` so the changes here are related to that